net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/yaddns/yaddns-$portVersion.tar.gz"
+CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+SOURCE_DIR="yaddns-yaddns-$portVersion"
```

Comment:
```diff
-CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+CHECKSUM_SHA256="ac220d767d165a8874d1fca1e369746ecb07338e333ae104b7b282a52f7630eb"
-SOURCE_DIR="yaddns-yaddns-$portVersion"
```

Reply:
Hmm, since we need the `configure.in` which is not available on the tarball from SourceForge, we need to switch back to the tarball from GitHub.